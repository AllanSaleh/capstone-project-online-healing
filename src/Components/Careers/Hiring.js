import React from 'react';

export default function Hiring() {
  return (
    <div className="bg-yellow-primary px-sides h-2/5 flex flex-col justify-evenly">
      <div className="text-title">OUR HIRING PHILOSOPHY</div>
      <div className="text-subtitle text-gray-600">
        To build the very best SEO tools, we need a workforce that reflects the diversity of our
        communities and customers. We want Online Therapy to be a place where everyone feels welcome
        and engaged, bar none. It's our mission and promise to interview a diverse and
        representative slate of candidates before making an offer for our open roles.
      </div>
    </div>
  );
}
