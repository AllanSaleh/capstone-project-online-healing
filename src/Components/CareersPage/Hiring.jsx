import React from 'react';
import { useTranslation } from 'react-i18next';

export default function Hiring() {
  const { t } = useTranslation();

  return (
    <div className="bg-yellow-primary px-sides h-2/5 flex flex-col justify-evenly py-4">
      <div className="text-3xl lg:text-title">{t('Open.Hiring')}</div>
      <div className="text-xl lg:text-subtitle text-gray-600 mt-4 text-justify">
        {t('Open.HiringIntro')}
      </div>
    </div>
  );
}
