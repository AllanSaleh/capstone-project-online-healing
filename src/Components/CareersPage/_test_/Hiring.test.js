import * as React from 'react';
import Hiring from "../Hiring";
import * as renderer from "react-test-renderer";

it('Hiring renders correctly', () => {
  const tree = renderer
    .create(<Hiring/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});