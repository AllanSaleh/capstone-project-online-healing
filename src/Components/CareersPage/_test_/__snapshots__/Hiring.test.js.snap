// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-yellow-primary px-sides h-2/5 flex flex-col justify-evenly py-4"
>
  <div
    className="text-3xl lg:text-title"
  >
    OUR HIRING PHILOSOPHY
  </div>
  <div
    className="text-xl lg:text-subtitle text-gray-600 mt-4 text-justify"
  >
    To build the very best SEO tools, we need a workforce that reflects the diversity of our communities and customers. We want Online Therapy to be a place where everyone feels welcome and engaged, bar none. It's our mission and promise to interview a diverse and representative slate of candidates before making an offer for our open roles.
  </div>
</div>
`;
